migrate.go

```diff
@@ -156,7 +156,7 @@ func collectMigrationsFS(fsys fs.FS, dirpath string, current, target int64) (Mig
 	for _, file := range sqlMigrationFiles {
 		v, err := NumericComponent(file)
 		if err != nil {
-			return nil, err
+			return nil, fmt.Errorf("Could not parse SQL migration file %q: %w", file, err)
```

Comment:
nit. Errors in go are typically lowercased.

Ref: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Can you update this and the other one?